lang/en.json

```
@@ -1090,6 +1090,59 @@
   }
 },
 
+"DND5E.CALENDAR": {
+  "Action": {
+    "AdvanceTime": "Advance by {amount}",
+    "OpenCharacterSheet": "Open Character Sheet",
+    "OpenPartySheet": "Open Party Sheet",
+    "ReverseTime": "Reverse by {amount}"
+  },
+  "Configuration": {
+    "DisabledMessage": "The DM must enable the calendar interface before it can be used by players.",
+    "Hint": "Control whether the calendar HUD is displayed and how it functions.",
+    "Label": "Configure Calendar",
+    "Name": "Calendar",
+    "Preferences": "Calendar Preferences"
+  },
+  "FIELDS": {
+    "enabled": {
+      "hint": "Allow the calendar UI to be used by all users.",
```

'used' perhaps might imply that players could use the other functions of the calendar, such as advancing time. I would suggest something like this, but maybe there is a better suggestion.


```suggestion
      "hint": "Allow the calendar UI to be viewed & configured by all users.",
```